src/borg/helpers/parseformat.py

```diff
@@ -564,7 +596,7 @@ def host(self):
     def canonical_path(self):
         if self.proto in ("file", "socket"):
             return self.path
-        if self.proto == "rclone":
+        if self.proto in ("rclone", "s3"):
             return f"{self.proto}:{self.path}"
```

Comment:
the "ssh" case includes much more stuff into the returned value. guess s3 is more similar to that.